patch.sh

```diff
@@ -0,0 +1,5 @@
+#/bin/bash
+
+# The 1..45 line is way off
+grep -v '45;1;all;Woodfall Rd;Middlesex;MA;02478' 25017.csv > 25017.csv.tmp
```

Comment:
This seems an odd way to go about fixing the data. What is the problem here? Can we add a generic sanity check to the csv conversion instead which drops the offending line?

Reply:
This isn't Woodfall Rd: https://wktmap.com/?38df26f8 and I made a manual SQL change after importing the 2023 data. I don't know how to automate that.

Reply:
Filtering linestrings that are too short?